test/cypress/support/command/command.js

```diff
@@ -12,6 +14,24 @@ Cypress.Commands.add('getByTestId', (selector, ...args) => cy.get(`[${TEST_ID_KE
  */
 Cypress.Commands.add('getById', (selector, ...args) => cy.get(`[id=${selector}]`, ...args));
 
+/**
+ *  Get input elements by name
+ *  @param selector - The name of the DOM elements
+ */
+Cypress.Commands.add('getInputByName', (name, ...args) => cy.get(`input[name=${escapeRegExp(name)}]`, ...args));
+
+/**
+ *  Get Typeahead Option by aria label
+ *  @param value - The value of the aria label
+ */
+Cypress.Commands.add('getTypeaheadOption', (value, ...args) => cy.get('.dropdown-item', ...args).contains(value));
```

Comment:
seems that it's not using ARIA label

Reply:
nope, this is the only way I found to access properly to the options without changing the typeahead component

Reply:
then please amend ```*  Get Typeahead Option by aria label
 *  @param value - The value of the aria label```

Reply:
Done